test/Operators/finitedifference/column.jl

```diff
@@ -87,6 +87,54 @@ end
     end
 end
 
+
+@testset "Scalar Field FiniteDifferenceSpaces - periodic" begin
```

Comment:
Thanks for adding this. Can we also add Mesh and Topology tests, in addition to Operators test?

Reply:
I've added Mesh tests. We don't really use any of the Topology properties.